source/OutfitterPanel.cpp

```diff
@@ -441,6 +441,30 @@ ShopPanel::BuyResult OutfitterPanel::CanBuy(bool onlyOwned) const
 				+ Format::CargoString(engineNeeded, "engine space") + ", and this ship has "
 				+ Format::MassString(engineSpace) + " free.";
 
+		int afterburnerSlotNeeded = -selectedOutfit->Get("afterburner slot");
+		int afterburnerSlotFree = playerShip->Attributes().Get("afterburner slot");
+		if(afterburnerSlotNeeded && !afterburnerSlotFree)
+			return "This afterbuner is designed to be installed in an afterburner slot, "
```

Comment:
I'd just say "dedicated slot" instead of repeating the word twice since it's the exact same name? idk much about wording though

Reply:
Precise or not, I would agree with "dedicated slot." Repetition might add value but it also makes the sentence clunky. And now that I've seen it, I can't unsee "This afterburner is designed to be installed in an _afterburner_ slot, ~~you dummy~~."

Reply:
lol.